versions.props

```diff
@@ -28,6 +28,8 @@ org.scala-lang.modules:scala-collection-compat_2.12 = 2.6.0
 org.scala-lang.modules:scala-collection-compat_2.13 = 2.6.0
 com.emc.ecs:object-client-bundle = 3.3.2
 org.immutables:value = 2.9.2
+commons-dbutils:commons-dbutils = 1.7
```

Comment:
Iceberg avoids runtime dependencies on Apache Commons. What is this used for? Can we remove it easily?

Reply:
Just QueryRunner with its statement-configuration/parameter-handling functionality along with the ResultSetHandler interface. Turned out to be surprisingly easy to replace the whole dbutils dependency with analogous but significantly simplified interfaces inline in JdbcSnowflakeClient.java, especially since we're only using String parameters in PreparedStatements for now and we probably don't need to worry about some of the more advanced dbutils configuration of things like maxFieldSize, fetchSize, etc. for now.